Video to PDF/script.py

```diff
@@ -0,0 +1,114 @@
+from tkinter import *
+from moviepy.editor import VideoFileClip
+from moviepy.editor import AudioFileClip
+from tkinter import filedialog
+from tkinter import messagebox
+from tkinter import ttk
+from fpdf import FPDF
+import threading
+import speech_recognition as sr
+import os
+
+
+#variables
+video_clip = ''
+audio_clip = ''
+
+#function to get video
+def get_video():
+    global video_filepath, video_clip
+    try:
+        video_filepath.set(filedialog.askopenfilename(title="Select your video file"))
+        video_clip = VideoFileClip(str(video_filepath.get()))
+    except:
+        messagebox.showerror("Error", "No video selected")
+
+
+#function to convert audio to pdf
+def audio_to_pdf():
+    global audio_clip
+    try :
+        audio_clip = video_clip.audio.write_audiofile(r"my_audio.wav")
+        r = sr.Recognizer()
+        with sr.AudioFile("my_audio.wav") as source:
+            audio_data = r.record(source)
+            text = r.recognize_google(audio_data)
+            write_file = open('my_text.txt', 'w')
+            write_file.write(text)
+            write_file.close()
+            text_to_pdf('my_text.txt')
+        messagebox.showinfo("Message", "Conversion Successfull")
+    except :
+        messagebox.showerror( "Error", "Conversion not performed")
+
+    video_filepath.set('')
+
+    progress_bar['value'] = 0
+    progress_bar.stop()
+
+    os.remove("my_audio.wav")
+    os.remove("my_text.txt")
+
+#function to convert text to pdf
+def text_to_pdf(file):
+    pdf = FPDF(format='letter', unit='in')
+    pdf.add_page()
+    pdf.set_font("Arial", size = 12)
+    effective_page_width = pdf.w - 2*pdf.l_margin
+
+    f = open(file, "r")
+
+    for x in f:
+        pdf.multi_cell(effective_page_width, 0.15, x)
+        pdf.ln(0.5)
+
+
+    pdf.output("my_pdf.pdf")
```

Comment:
Add this path:
```suggestion
    pdf.output("./ Video to PDF/my_pdf.pdf")
```
Also, the file name needs to be the same as the video file name with added (pdf). For instance,
input: video12.mp4
output: video12 (pdf).pdf

Reply:
I have added the path now...
but for name of video and pdf to be same I tried writing ``pdf.output(video_filepath.split('.')[0]+"(pdf)"+".pdf")`` but it isn't working....

Reply:
Checkout this PR implementation: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/1105

Reply:
like this?

filename = os.path.basename(video_filepath)
pdf.output("./Video to PDF/" + filename.split('.')[0] + "(pdf)" + ".pdf")